trading/analysis/technical/weekly_indicators/bin/compute_ema.ml

```diff
@@ -0,0 +1,32 @@
+open Core
+open Weekly_indicators.Ema
+
+let parse_file filename =
```

Comment:
document how this fail, e.g. if one line failed to parse, would the whole function fail?